app/src/main/java/com/framgia/englishconversation/screen/imageDetail/ImageDetailViewModel.java

```diff
@@ -18,8 +22,20 @@
     private TimelineModel mTimelineModel;
     private FragmentManager mManager;
     private Fragment mFragment;
+    private Context mContext;
+    private OnMediaModelItemTouchListener<List<MediaModel>> mListener =
+            new OnMediaModelItemTouchListener<List<MediaModel>>() {
 
-    public ImageDetailViewModel(TimelineModel timelineModel, FragmentManager manager) {
+                @Override
+                public void onTouchListener(List<MediaModel> items, int postion) {
+                    mContext.startActivity(
```

Comment:
ERROR: 'method def' child have incorrect indentation level 20, expected level should be one of the following: 12, 16.